test/utils/images/images.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import "os"
```

Comment:
ok, so you moved the `PauseImage` const in the `test/utils/images` package. I don't feel strongly about that. But then you need to either revert the change or to import the package here...

Reply:
importing `test/utils/objects` caused cyclic import which isn't allowed in Go.